### DISCUSSION TITLE
Cookie stick forever on FF

### DISCUSSION THREAD

**Discussion by akasaka99** [Cookie-AutoDelete/Cookie-AutoDelete#1167] [Q&A / Support] [answered]
Hi, CAD does not seem to work all anymore, does not delete cookie on closing tab on FF?? 
Maybe it's because I'm using FF Containers (though before it was working fine... ), is there anything in the setup I should look at?
Anyone else noticed CAD became totally ineffective at cleaning cookies  (it works fine on Brave)
Thanks

**Comment by kennethtran93** [comment, marked as answer]
See #807.  In short, if your privacy setting is 'strict' or the custom cookie setting is set to 'cross-site cookies', then that breaks all web extensions handling cookies as there is no API to handle the new variables created with the new protection system.
